src/clj/ataru/haku/haku_service.clj

```diff
@@ -4,31 +4,53 @@
    [ataru.applications.application-store :as application-store]
    [ataru.forms.form-store :as form-store]))
 
-(defn- raw-haku-row->hakukohde [tarjonta-service raw-haku-row]
-  (merge (select-keys raw-haku-row [:application-count :processed :haku])
-         {:oid  (:hakukohde raw-haku-row)
-          :name (or
-                  (.get-hakukohde-name tarjonta-service (:hakukohde raw-haku-row))
-                  {:fi (:hakukohde raw-haku-row)})}))
+(defn- raw-haku-row->hakukohde
+  [tarjonta-service raw-haku-row]
```

Comment:
`[tarjonta-service {:keys [hakukohde application-count processed processing] :as raw-haku-row}]` pitäis ajaa saman asian kuin tuo erillinen `let`.